web-api/src/personalization/routes.rs

```diff
@@ -356,6 +355,58 @@ struct UnvalidatedSemanticSearchQuery {
     personalize: Option<UnvalidatedPersonalize>,
     #[serde(default)]
     enable_hybrid_search: bool,
+    // TODO[pmk/now] to we keep enable_hybrid_search
+    #[serde(default, rename = "_dev")]
+    dev: Option<DevOptions>,
+}
+
+#[derive(Debug, Default, Deserialize)]
+#[serde(default)]
+struct DevOptions {
+    hybrid: Option<HybridDevOption>,
+    use_es_rrf: bool,
+}
+
+#[derive(Debug, Deserialize)]
+struct HybridDevOption {
+    normalize_knn: NormalizationFn,
+    normalize_bm25: NormalizationFn,
+    merge_fn: MergeFn,
+}
```

Comment:
But there are potentially other future dev options. We could have such an enum under `_dev.hybrid` but then it would be `_dev.hybrid.hybrid` etc. so idk.

Reply:
i agree, an enum would be better for now, but we may need to change it again later. up to you to decide.

Reply:
I made `_dev.hybrid` an enum so it's  `_dev.hybrid.customize` and `_dev.hybrid.es_rrf`, I also allowed specified the `rank_constant` in case of elastic search RRF (you already can do so for dev hybrid search in case of use the rrf merge function).